lib/datadog/core/telemetry/v1/profiler.rb

```diff
@@ -14,6 +14,12 @@ def initialize(version:)
 
             @version = version
           end
+
+          def to_h
+            {
+              version: @version
+            }
+          end
         end
```

Comment:
I don't quite understand why is this needed? 🤔 

There's a single dd-trace-rb library, so this version will always be the same as the library version.

Reply:
The telemetry API expects the appsec/profiler version to be passed in in the products object: https://github.com/DataDog/instrumentation-telemetry-api-docs/blob/main/GeneratedDocumentation/ApiDocs/v1/SchemaDocumentation/Schemas/profiler.md

However since it's just one field, do you think it'd be better to remove the `AppSec` and `Profiler` classes and just use a regular Hash instead?